### PR TITLE
Update blissfuljs homepage & add blissfuljs@1.0.5 w/ npm auto-update

### PR DESCRIPTION
**Pull request by kirintwn**
**Library name:** blissfuljs
**Git repository url:** https://github.com/LeaVerou/bliss/

I have done the following updates.
- commit `e687d30`
  - changed `homepage` in package.json from `http` to `https` protocol.
- commit `01de0b4`
  - changed blissfuljs auto-update config from git to npm. (Ref: https://github.com/LeaVerou/bliss/issues/215)
- commit `addf0be`
  - manually added version 1.0.5 (grabbed missing files from their [homepage](https://blissfuljs.com/))

Pull request for issue: #12318

### CODE REVIEW

File: ajax/libs/blissfuljs/package.json [hunk 1]
```diff
@@ -17,19 +17,16 @@
...
-        "basePath": "",
-        "files": [
-          "bliss.js",
-          "bliss.min.js",
-          "bliss.shy*.js"
-        ]
-      }
-    ]
-  }
+  "homepage": "https://blissfuljs.com",
```
Comment (extend1994) [inline ajax/libs/blissfuljs/package.json]: Let's do this change using another commit because it's meta-data.
Please notice this commit should be done before new version add & auto-update.
You may use `reset --soft HEAD^` to reserve your changes in unstaged area and them redo commits.
Here is a suggestion for first commit:
```
Add blissfuljs@1.0.5 w/ npm auto-update
close #12318
```

second commit
```
Update blissfuljs homepage to use https, cc #12318
```

Refer https://github.com/cdnjs/cdnjs/pull/11551#pullrequestreview-59526021

File: ajax/libs/blissfuljs/package.json [hunk 2]
```diff
@@ -17,19 +17,16 @@
...
+  "npmName": "blissfuljs",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "bliss.js",
+        "bliss.min.js",
+        "bliss.shy*.js"
```
Comment (extend1994) [inline ajax/libs/blissfuljs/package.json]: Let's simplified L26-L28 to get the same files.

**Comment by cy91244** [comment]
@extend1994 Thanks for your quick response♡
I have modified the PR as you suggested, please confirm the changes.

File: ajax/libs/blissfuljs/package.json [hunk 3]
```diff
@@ -18,18 +18,13 @@
...
-        ]
-      }
-    ]
-  }
+  "npmName": "blissfuljs",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "bliss?(.shy)?(.min).js"
```
Comment (extend1994) [inline ajax/libs/blissfuljs/package.json]: Let's be use a more flexible pattern `bliss!(._.)*` to just exclude possible file `bliss._.js`.

**Comment by cy91244** [comment]
@extend1994 I have changed the file pattern from `bliss?(.shy)?(.min).js` to `bliss!(._.)*`.
Please confirm the changes, thank you!

**Comment by sashberd** [comment]
@extend1994 I re-add the files from npm. FYR

**Comment by sashberd** [comment]
@extend1994 ping

**Comment by extend1994** [comment]
@sashberd Sorry for the late reply, did you minify them with [web-minify-helper](https://github.com/PeterDaveHello/web-minify-helper)?

**Comment by sashberd** [comment]
@extend1994 I updated files with minifier. FYR

File: ajax/libs/blissfuljs/1.0.5/bliss.min.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
+,// Properties with special handling in classes classProps:{// Lazily evaluated properties
```
Comment (extend1994) [inline ajax/libs/blissfuljs/1.0.5/bliss.min.js]: It's obviously not minified.

File: ajax/libs/blissfuljs/1.0.5/bliss.shy.min.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
+,// Properties with special handling in classes classProps:{// Lazily evaluated properties
```
Comment (extend1994) [inline ajax/libs/blissfuljs/1.0.5/bliss.shy.min.js]: Neither is it.

**Comment by sashberd** [comment]
@extend1994 But the minified files were added in previous PR and you told me to minify them with our minifier. So what is the right way? Look as you see this is not my PR and I just trying to help with this one. Please provide solution that will satisfy you review to this PR. Thanks

**Comment by extend1994** [comment]
@sashberd For v1.0.5, I think downloading from its homepage is good enough, thanks!

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR
